Source/Core/isArray.js

```diff
@@ -6,6 +6,8 @@ import defined from './defined.js';
      *
      * @param {*} value The value to test.
      * @returns {Boolean} true if the value is an array, false otherwise.
+     * @deprecated See https://github.com/AnalyticalGraphicsInc/cesium/issues/8526.
+     * Use `Array.isArray` instead
      */
     var isArray = Array.isArray;
```

Comment:
Make a call to `deprecationWarning` in this function so something gets logged to the console

Reply:
See ab1edca. I had to create a function to display the message; otherwise it was always shown (isArray is imported in Cesium.js)